mrp_workcenter_hierarchical/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Use short headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py